# HG changeset patch
# User Rik <rik@octave.org>
# Date 1520008906 28800
#      Fri Mar 02 08:41:46 2018 -0800
# Node ID cce5d2ea39ff6804744bde3a1fbdafae78f6859b
# Parent  53f68d0fb1224d838d0726403ce6b7604724f864
acinclude.m4: Fix typo in previous cset (53f68d0fb122).

* acinclude.m4: Fix typo in AC_DEFINE name
QOBJECT_FINDCHILDREN_ACCEPTS_FINDCHILDOPTIONS.

diff --git a/m4/acinclude.m4 b/m4/acinclude.m4
--- a/m4/acinclude.m4
+++ b/m4/acinclude.m4
@@ -440,17 +440,17 @@ AC_DEFUN([OCTAVE_CHECK_FUNC_QOBJECT_FIND
         ]])],
       octave_cv_func_qobject_findchildren_accepts_findchildoptions=yes,
       octave_cv_func_qobject_findchildren_accepts_findchildoptions=no)
     CPPFLAGS="$ac_octave_save_CPPFLAGS"
     CXXFLAGS="$ac_octave_save_CXXFLAGS"
     AC_LANG_POP(C++)
   ])
   if test $octave_cv_func_findchildren_accepts_findchildoptions = yes; then
-    AC_DEFINE(QOBJECT_FINDCHILDREN_ACCEPTS_QFINDCHILDOPTIONS, 1,
+    AC_DEFINE(QOBJECT_FINDCHILDREN_ACCEPTS_FINDCHILDOPTIONS, 1,
       [Define to 1 if 'QObject::findChildren' accepts 'Qt::FindChildOptions' argument.])
   fi
 ])
 dnl
 dnl Check whether the Qt class QTabWidget has the setMovable member function.
 dnl This member function was introduced in Qt 4.5.
 dnl
 dnl FIXME: Delete this entirely when we can safely assume that Qt 4.5 or later
